vslib/SwitchStateBase.cpp

```diff
@@ -4294,3 +4328,61 @@ void SwitchStateBase::send_tam_tel_type_config_change(
 
     m_switchConfig->m_eventQueue->enqueue(std::make_shared<Event>(EVENT_TYPE_NOTIFICATION, payload));
 }
+
+void SwitchStateBase::generate_ipfix_templates(sai_object_id_t tam_tel_type_id)
```

Comment:
function signature not matching other functions

Reply:
also this function naming suggest that it will return something

Reply:
Done